### PR TITLE
[BIP-749] Grant `add_reward` Permissions to Maxis’ Omnichain Multisig

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**

### CODE REVIEW

**Comment by gosuto-inzasheru** [comment]
still needs other chains

File: BIPs/2024-W52/BIP-749.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "bytes32", "name": "role", "type": "bytes32" },
+          { "internalType": "address", "name": "account", "type": "address" }
+        ],
+        "name": "grantRole",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "role": "0x3bf29175652a3f0fac5abb715d0b7fe2e7b597e2e2eff555dac6b21a20a7c83e",
```
Comment (Xeonus) [inline BIPs/2024-W52/BIP-749.json:27]: Shouldn't we use gauge-factory-v2? See [deployments](https://github.com/balancer/balancer-deployments/blob/e438df9dc2b9cb9f5ee31d34938d4ca58e942fb7/action-ids/mainnet/action-ids.json#L714)

**Review by Xeonus — COMMENTED** [review]
- Correctly grants role as confirmed in deployments repo (example: [arbitrum](https://github.com/balancer/balancer-deployments/blob/e438df9dc2b9cb9f5ee31d34938d4ca58e942fb7/action-ids/arbitrum/action-ids.json#L620))
- correctly uses child-chain-factory-v2 on other chains

File: BIPs/2024-W52/BIP-749.json (hunk 1 above)
Comment (gosuto-inzasheru) [reply BIPs/2024-W52/BIP-749.json:27]: same same but different

Comment (Xeonus) [reply BIPs/2024-W52/BIP-749.json:27]: Confirming role is identical. So no change needed. Thanks

**Review by Xeonus — APPROVED** [review]

**Pull request merged**